data_quality/datapackage.json

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
For the internal file in the CLI, might be better to call it `datapackage.json.template` or something similar.